src/material/timepicker/timepicker-input.ts

```diff
@@ -42,6 +42,7 @@ import {Subscription} from 'rxjs';
 import {DOWN_ARROW, ESCAPE, hasModifierKey, UP_ARROW} from '@angular/cdk/keycodes';
 import {validateAdapter} from './util';
 import {DOCUMENT} from '@angular/common';
+import {_getFocusedElementPierceShadowDom} from '../../cdk/platform';
```

Comment:
The CI is failing because of this import. Can you change it to be `@angular/cdk/platform`?